notebooks/7-rotation-speeds/interm_rotation_rate.ipynb

```diff
@@ -0,0 +1,2926 @@
+{
```

Comment:
<p>Not that libretexts chemistry is a bad resource, but can we find a more topical one? There's a <a href="https://docs.lightkurve.org/tutorials/3-science-examples/asteroseismology-oscillating-star-periodogram.html" target="_blank">lighkurve tutorial that somewhat touches on the topic</a>, but maybe isn't the best link. They do have a good intuitive explanation of why we need this though:</p><p> </p><blockquote>The Nyquist frequency is a property of a time series that describes the maximum frequency that can be reliably determined in a periodogram. It stems from the assumption that you need a minimum of two observations per oscillation period to observe a pattern (one observation on the “up,” and one on the “down” oscillation). It is defined as follows:</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>